Core/include/Acts/Seeding/GNN_Geometry.hpp

```diff
@@ -0,0 +1,369 @@
+
+namespace Acts {
+class TrigInDetSiLayer {
+ public:
+  int m_subdet;//combined ID  
+  int m_type;//0: barrel, +/-n : endcap
+  float m_refCoord;
+  float m_minBound, m_maxBound;
+
+  TrigInDetSiLayer(int subdet, short int type, float center, float min, float max ) : m_subdet(subdet), m_type(type), m_refCoord(center), m_minBound(min), m_maxBound(max) {
+
+  } 
+};
+
+template <typename space_point_t>  
+class TrigFTF_GNN_Layer {
+ public:
+
+  TrigFTF_GNN_Layer(const TrigInDetSiLayer& ls, float ew, int bin0) : m_layer(ls), m_etaBinWidth(ew) {
+
+    if(m_layer.m_type == 0) {//barrel
+      m_r1 = m_layer.m_refCoord;
+      m_r2 = m_layer.m_refCoord;
+      m_z1 = m_layer.m_minBound;
+      m_z2 = m_layer.m_maxBound;
+    }
+    else {//endcap
+      m_r1 = m_layer.m_minBound;
+      m_r2 = m_layer.m_maxBound;
+      m_z1 = m_layer.m_refCoord;
+      m_z2 = m_layer.m_refCoord;
+    }
+
+    float t1   = m_z1/m_r1;
+    float eta1 = -std::log(sqrt(1+t1*t1)-t1);
+
+
+    float t2   = m_z2/m_r2;
+    float eta2 = -std::log(sqrt(1+t2*t2)-t2);
+
+    m_minEta = eta1;
+    m_maxEta = eta2;
+    
+    if(m_maxEta < m_minEta) {
+      m_minEta = eta2;
+      m_maxEta = eta1;
+    }
+
+    m_maxEta += 1e-6;//increasing them slightly to avoid range_check exceptions
+    m_minEta -= 1e-6;
+
+    float deltaEta = m_maxEta - m_minEta;
+
+    int binCounter = bin0;
+
+    if(deltaEta < m_etaBinWidth) {
+      m_nBins = 1;
+      m_bins.push_back(binCounter++);
+      m_etaBin = deltaEta;
+      if(m_layer.m_type == 0) {//barrel
+        m_minRadius.push_back(m_layer.m_refCoord - 2.0);
+        m_maxRadius.push_back(m_layer.m_refCoord + 2.0);
+        m_minBinCoord.push_back(m_layer.m_minBound);
+        m_maxBinCoord.push_back(m_layer.m_maxBound);
+      }
+      else {//endcap
+        m_minRadius.push_back(m_layer.m_minBound - 2.0);
+        m_maxRadius.push_back(m_layer.m_maxBound + 2.0);
+        m_minBinCoord.push_back(m_layer.m_minBound);
+        m_maxBinCoord.push_back(m_layer.m_maxBound);
+      }
+    }
+    else {
+      float nB = static_cast<int>(deltaEta/m_etaBinWidth);
+      m_nBins = nB;
+      if(deltaEta - m_etaBinWidth*nB > 0.5*m_etaBinWidth) m_nBins++;
+
+      m_etaBin = deltaEta/m_nBins;
+
+      if(m_nBins == 1) {
+        m_bins.push_back(binCounter++);
+        if(m_layer.m_type == 0) {//barrel
+    m_minRadius.push_back(m_layer.m_refCoord - 2.0);
+          m_maxRadius.push_back(m_layer.m_refCoord + 2.0);
+    m_minBinCoord.push_back(m_layer.m_minBound);
+    m_maxBinCoord.push_back(m_layer.m_maxBound);
+        }
+        else {//endcap
+    m_minRadius.push_back(m_layer.m_minBound - 2.0);
+          m_maxRadius.push_back(m_layer.m_maxBound + 2.0);
+    m_minBinCoord.push_back(m_layer.m_minBound);
+    m_maxBinCoord.push_back(m_layer.m_maxBound);
+        }
+      }
+      else {
+
+        float eta = m_minEta+0.5*m_etaBin;
+
+        for(int i=1;i<=m_nBins;i++) {
+
+    m_bins.push_back(binCounter++);
```

Comment:
There seems to be some funny indentation here and elsewhere. Is this copied from the original (in which case it could be fixed later) or some funny copy/paste? Perhaps it was due to different tab expansion.

Probably the easiest fix would be to run `clang-format` on the new files. That can be set up in VS Code, or there are instructions in the ACTS docs. This could be left until later when the MR is complete.